pygmt/datasets/earth_relief.py

```diff
@@ -46,13 +46,13 @@ def load_earth_relief(resolution="01d", region=None, registration=None, use_srtm
     registration : str
         Grid registration type. Either ``pixel`` for pixel registration or
         ``gridline`` for gridline registration. Default is ``None``, where
-        a pixel-registered grid is returned unless only the
-        gridline-registered grid is available.
+        a gridline-registered grid is returned unless only the pixel-registered
+        grid is available.
```

Comment:
The behavior changes between GMT 6.3 and 6.4. Since we're going to support multiple GMT versions (https://github.com/GenericMappingTools/pygmt/issues/1991), perhaps we should document the behaviors in both GMT 6.3 and 6.4 here?